src/backend/api/reviews.js

```diff
@@ -0,0 +1,68 @@
+const express = require("express");
+const { on } = require("../database");
+const router = express.Router();
+const knex = require("../database");
+
+
+router.get("/", async (request, response) => {
+    const reviews = await knex("reviews");
+    response.json(reviews);
+})
+
+router.post("/", async (request, response) => {
+
+    const insertedReviews = await knex("reviews")
```

Comment:
would recommend to use a try catch block where you capture the error in case the insert fails and return an response to the client as an error code